angel_system/data/medical/load_bbn_data.py

```diff
@@ -365,7 +401,8 @@ def bbn_activity_txt_to_csv(task, root_dir, output_dir, label_version):
 
                 track_id += 1
         csv_f.close()
-
+        
+    return used_videos
```

Comment:
This looks like a new return statement for this function. Please update the doc-string accordingly.